_episodes/08-conflict.md

```diff
@@ -0,0 +1,338 @@
+---
+title: Conflicts
+teaching: 15
+exercises: 0
+questions:
+- "What do I do when my changes conflict?"
+objectives:
+- "Explain what conflicts are and when they can occur."
+- "Understand how to resolve conflicts resulting from a merge."
+keypoints:
+- "Conflicts occur when files are changed in the same place in two commits that are being merged."
+- "The version control system does not allow one to overwrite changes blindly during a merge, but highlights conflicts so that they can be resolved."
+---
+
+If we are working on a piece of software on both our laptop and a
+server in the lab, we could make different changes to each copy.
+Version control helps us manage these [conflicts]({{ page.root}}/reference/#conflicts)
+by giving us tools to [resolve]({{ page.root}}/reference/#resolve) overlapping changes.
+
+To see how we can resolve conflicts, we must first create one.  The
+file `mars.txt` currently looks like this in our `planets` repository:
+
+~~~
+$ cat mars.txt
+~~~
+{: .bash}
+
+~~~
+Cold and dry, but everything is my favorite color
+The two moons may be a problem for Wolfman
+But the Mummy will appreciate the lack of humidity
+~~~
+{: .output}
+
+Let's create a new branch for discussing Mars' temperature and
+checkout that branch.
+
+~~~
+$ git branch marsTemp
+~~~
+{: .bash}
+
+But before we make changes related to Mars' temperature,
+let's add a line to Mars.txt.
+
+~~~
+$ nano mars.txt
+$ cat mars.txt
+~~~
+{: .bash}
+
+~~~
+Cold and dry, but everything is my favorite color
+The two moons may be a problem for Wolfman
+But the Mummy will appreciate the lack of humidity
+I'll be able to get 40 extra minutes of beauty rest
+~~~
+{: .output}
+
+and commit that change to the `master` branch
+
+~~~
+$ git add mars.txt
+$ git commit -m "Add a line about the daylight on Mars."
+~~~
+{: .bash}
+
+~~~
+[master 5ae9631] Add a line in our home copy
+ 1 file changed, 1 insertion(+)
+~~~
+{: .output}
+
```

Comment:
That's a good call. Can do the same after adding the change in the `marsTemp` branch.

To make sure we're on the same page, `git branch` was meant to be `git status`?